deploy/charts/cert-manager/values.yaml

```diff
@@ -8,6 +8,11 @@ global:
   imagePullSecrets: []
   # - name: "image-pull-secret"
 
+  ## Labels to apply to all resources
```

Comment:
Please add a note that this does not add labels to the resources created dynamically by the controllers and provide at least these examples:
eg. podTemplate/ ingressTemplate in ACMEChallengeSolverHTTP01Ingress
https://cert-manager.io/docs/reference/api-docs/#acme.cert-manager.io/v1.ACMEChallengeSolverHTTP01Ingress
eg. secretTemplate in CertificateSpec
https://cert-manager.io/docs/reference/api-docs/#cert-manager.io/v1.CertificateSpec